src/assets/styles/_forms.scss

```diff
@@ -58,10 +63,47 @@
   font-size: $kui-font-size-50;
 }
 
+.field-group-list > *+* {
```

Comment:
nit/question: Do you know if there a lint rule to always give us spaces around these things?

```css
.field-group-list > * + * {
}

```

Reply:
I don’t.

Reply:
I made an issue https://github.com/kumahq/kuma-gui/issues/1484